locale/ja/site.json

```
@@ -6,13 +6,13 @@
   "locale": "ja",
   "language": "日本語",
   "languageEnglishVersion": "Japanese",
-  "scrollToTop": "上部へスクロールする",
+  "scrollToTop": "ページの先頭へ",
   "reportNodeIssue": "Node.js の不具合を報告",
   "reportWebsiteIssue": "サイトの不具合を報告",
   "getHelpIssue": "ヘルプ",
   "by": "by",
-  "all-downloads": "All download options",
-  "nightly": "Nightly builds",
+  "all-downloads": "ダウンロード全種一覧",
+  "nightly": "最新不安定版",
```

不安定とは限らないと思うので、「開発版」でどうでしょうか。
```suggestion
  "nightly": "最新の開発版",
```

---

確かにそちらの方が適切ですね。